resources/js/ingame/e7c74974620fa35b197315ebdbb8c2.js

```diff
@@ -49092,6 +49092,8 @@ function getEmptySlotActions(galaxyContentObject, systemData) {
   let coloniseMission = galaxyContentObject.availableMissions.find(availMission => availMission.missionType === 7);
   let colonisationLink = "";
 
+  console.log(systemData.canColonize, coloniseMission, coloniseMission.link);
```

Comment:
Can you remove the console log on this line?

Reply:
Oh yes good catch!